pegasus/sites.v3/hourofcode.com/views/how_to_nav.haml

```diff
@@ -1,28 +1,28 @@
-%div.resources-navigation
-  %a.nav-link{href: resolve_url('/how-to')}
-    %div.nav-header= hoc_s(:resources_how_to_guides)
-  %div.nav-item
-    %a.nav-link{href: resolve_url('/how-to')}
-      = hoc_s(:how_to_nav_educators)
-  %div.nav-item
-    %a.nav-link{href: resolve_url('/how-to/afterschool')}
-      = hoc_s(:how_to_nav_afterschool_educators)
-  %div.nav-item
-    %a.nav-link{href: resolve_url('/how-to/companies')}
-      = hoc_s(:how_to_nav_companies)
-  %div.nav-item
-    %a.nav-link{href: resolve_url('/how-to/districts')}
-      = hoc_s(:how_to_nav_districts)
-  %div.nav-item
-    %a.nav-link{href: resolve_url('/how-to/events')}
-      = hoc_s(:how_to_nav_assemblies)
-  %div.nav-item
-    %a.nav-link{href: resolve_url('/how-to/parents')}
-      = hoc_s(:how_to_nav_parents)
-  %div.nav-item
-    %a.nav-link{href: resolve_url('/how-to/public-officials')}
-      = hoc_s(:how_to_nav_public_officials)
-  %div.nav-item
-    %a.nav-link{href: resolve_url('/how-to/volunteers')}
-      = hoc_s(:how_to_nav_volunteers)
-= view :resources_nav

```

Comment:
Most of these changes are just converting the file to use Unix line-endings.  Hide whitespace changes to see the substance of the change, if desired.